pkg/action/action_handler.go

```diff
@@ -22,11 +22,12 @@ import (
 )
 
 const (
-	defaultActionCacheTTL = time.Second * 50
+	defaultActionCacheTTL = time.Second * 100
 )
 
 type ActionHandlerConfig struct {
 	kubeClient     *client.Clientset
+	kubeletClient  *kubelet.KubeletClient
```

Comment:
Since not every action handler needs `kubeletClient`, so maybe to make `ActionHandlerConfig` an interface is a better idea.

Reply:
I think interface will not improve code readability, nor runtime performance.  Let's keep it simple.